sbin/prepareWorkspace.sh

```diff
@@ -515,7 +515,21 @@ prepareCacerts() {
     echo "Generating cacerts from Mozilla's bundle"
 
     cd "$SCRIPT_DIR/../security"
-    ./mk-cacerts.sh --keytool "${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/keytool"
+    # Our Solaris/SPARC box exhibits slowness when running keytool
+    # Therefore pull the cacerts file from the latest Solaris/x64 build
+    if [ "$ARCHITECTURE" = "sparcv9" ]; then
+      mkdir tmp
+      cd tmp || exit 1
+      wget -q -O jdk8build.tgz "https://api.adoptopenjdk.net/v3/binary/latest/8/ea/solaris/sparcv9/jre/hotspot/normal/adoptopenjdk?project=jdk"
```

Comment:
Are the certs bundle the same for each Java version?

Reply:
By Java version do you mean 8 vs 11 vs 15? I haven't confirmed (although I'd hope so - I imagine @aahlenst could give a quick answer!) but since we're only building JDK8 on Solaris/SPARC just now it isn't relevant for the purposes of this fix.